Assets/Scripts/UI/Menu/ImportingContributorsList.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ImportingContributorsList : MonoBehaviour {
```

Comment:
Please use a new line for your brackets. This is C#, not Java >.> (same goes for your Start method) and ScrollingScrollView class